extensions/amp-story/amp-story-player.md

```diff
@@ -0,0 +1,144 @@
+    <code>&lt;link href="https://cdn.ampproject.org/amp-story-player-v0.css" rel='stylesheet' type='text/css'></code>
+    </td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td><ul>
+      <li>See <a href="https://github.com/ampproject/amphtml/blob/master/examples/amp-story/player.html">code snippet</a>.</li>
+    </ul></td>
+  </tr>
+</table>
+
+[TOC]
+
+## Version notes
+
+| Version | Description      |
+| ------- | ---------------- |
+| 1.0     | Current version. |
+
+## AMP story player
+
+The `amp-story-player` allows you to embed and play your favorite stories in your own website.
+
+### Features
+
+- Improved experience over time (AKA the "Software as a Service" paradigm)
+- Prerendering (coming soon!)
+- Prefetching and caching resources (coming soon!)
+- Customizable embed UI / chrome (coming soon!)
+- Swipe/tap to next/previous (coming soon!)
+
+### Boilerplate
+
+The following markup is a decent starting point or boilerplate. Copy this and save it to a file with a `.html` extension.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script
+      async
+      src="https://cdn.ampproject.org/amp-story-player-v0.js"
+    ></script>
+    <link
+      href="https://cdn.ampproject.org/amp-story-player-v0.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+  </head>
+  <body>
+    <h1>This is a NON-AMP page that embeds a story below:</h1>
+    <amp-story-player style="width: 360px; height: 600px;">
+      <a
+        href="https://example.com/story.html"
+        style="--story-player-poster: url('https://example.com/img/promo3x4.jpg');"
+      >
+        <span>A description for my story.</span>
+      </a>
+    </amp-story-player>
+  </body>
+</html>
+```
+
+### API
+
+## `<amp-story-player>`
+
+The high-level API for the player looks like the following:
+
+```html
+<amp-story-player style="width: 360px; height: 600px;">
+  <a href="https://www.example.com/story1.html"></a>
+  <a href="https://www.example.com/story2.html"></a>
+  <a href="https://www.example.com/story3.html"></a>
+  <a href="https://www.example.com/story4.html"></a>
+</amp-story-player>
+```
+
+The inline width and height ensures that the player will not cause any jumps in your website while the script is being loaded. Feel free to modify the values, but we recommend mantaining a 3:5 aspect ratio.
+
+## `Children of <amp-story-player>`
+
+The `<amp-story-player>` component contains one or more `<a>` tags, each of which represent a story. Use the `href` attribute to point to a particular story.
+
+To provide a better experience for the user while the story is being loaded, as well as allowing search engines to crawl the embedded stories, you can provide some text inside the anchor tags.
+
+You can also provide an image to work as a placeholder while the story is being loaded. Just add a `--story-player-poster` CSS variable as an inline style of the first anchor tag with its corresponding image URL:
```

Comment:
This information should be moved under the first example. I'd update the title from `## Children of <amp-story-player>` to `### Specify embedded story`.

Reply:
Before I can dive deeper into this section I have some clarification questions: 

- Does the story need to be hosted somewhere else for it to be linked to? Where does this story need to live for the url to point to it? 
- It says `you can provide some text inside the anchor tags` then it demonstrates a `<span>` tag. Does text need to be defined in a `<span>` tag? Can I just put text? What about other text nodes like `<p>` or even `<div>`? Can I style them?
- Does this poster image need to match my stories cover image? Is there a way for me to link directly to the cover image?

Reply:
> Does the story need to be hosted somewhere else for it to be linked to? Where does this story need to live for the url to point to it?

It could be in the same site or another one, as long as accessible via the URL.

> It says you can provide some text inside the anchor tags then it demonstrates a <span> tag. Does text need to be defined in a <span> tag? Can I just put text? What about other text nodes like <p> or even <div>? Can I style them?

Good point, it does not, publishers can do whatever they want here. We don't have any validation since this isn't valid AMP. I'm going to remove the `<span>` to keep it simple in the example.

> Does this poster image need to match my stories cover image? Is there a way for me to link directly to the cover image?

There's no way for us to validate this, so you could in theory point to whatever image you wanted. We recommend that it matches the corresponding story's cover image, since this is what users will see before the story is completely loaded.